### PR TITLE
End of Book (Appendix + Index): Parts and Settings

### PR DESCRIPTION
**Pull request by clee1060**
https://github.com/openstax/cnx-recipes/issues/1154 Appendix
https://github.com/openstax/cnx-recipes/issues/1153 Index

### CODE REVIEW

File: styles/designs/carnival/parts/_index-spaces.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import 'index-components';
+
+// Superspaces
+//Learning global superspace
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_index-spaces.scss]: I think this comment can be removed?

File: styles/designs/carnival/parts/_appendix-spaces.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@import 'appendix-components';
+
+// Superspaces
+//Learning global superspace
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_appendix-spaces.scss]: I think this comment can be removed?

File: styles/designs/carnival/parts/_appendix-components.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+$AppendixLayerTwoComponent: (
+  _name: "Layer",
+  _subselector: ' > [data-depth="2"]',
+  _properties: (
+    color: enum('ValueSet:::NULLABLE'),
+    font-family: enum('ValueSet:::NULLABLE'),
+    font-size: enum('ValueSet:::NULLABLE'),
+    font-weight: normal,
+  )
+);
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_appendix-components.scss]:
```suggestion
);

```

File: styles/designs/carnival/parts/_index-components.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+$IndexPageNumberComponent: (
+  _name: "PageNumber",
+  _subselector: ' > .os-term-section-link',
+  _properties: (
+    content: enum('ValueSet:::NULLABLE'),
+    text-decoration: none,
+    pointer-events: none,
+    color: enum('ValueSet:::NULLABLE'),
+  )
+);
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_index-components.scss]:
```suggestion
);

```

File: styles/templates/template1/_typography.scss [hunk 5]
```diff
@@ -14,5 +14,6 @@
     baseLineHeight: 10px,
     baseFontSize: 10px,
     exampleFontSize: 1.44rem,
+    paragraphText: "Alegreya",
```
Comment (tomjw64) [inline styles/templates/template1/_typography.scss]: Is this font specifically for appendices only?

File: styles/templates/template1/parts/_appendix-settings.scss [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+            font-family: (_ref: 'typography:::paragraphText'),
+            font-size: not-implemented(1.44rem),
+        ),
+        SubTitle: (
+            color: (_ref: 'colorMap:::subsubtitleColor'),
+            font-family: (_ref: 'typography:::sansFont'),
+            font-size: not-implemented(2rem),
+        ),
+    ),
+));
```
Comment (tomjw64) [inline styles/templates/template1/parts/_appendix-settings.scss]:
```suggestion
));

```

File: styles/templates/template1/parts/_index-settings.scss [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+            font-family: (_ref: 'typography:::paragraphText'),
+            font-size: not-implemented(1.44rem),
+            padding-right: not-implemented(1rem),
+        ),
+        PageNumber: (
+            content: '""' target-counter(attr(href, url), page),
+            color: (_ref: 'colorMap:::fontBodyColor'),
+        ),
+    ),
+));
```
Comment (tomjw64) [inline styles/templates/template1/parts/_index-settings.scss]:
```suggestion
));

```

File: styles/templates/template1/_typography.scss (hunk 5 above)
Comment (clee1060) [reply styles/templates/template1/_typography.scss]: I don't think so?

Comment (helenemccarron) [reply styles/templates/template1/_typography.scss]: I also think naming of this element should be more specific, to me this seems like all paragraphs in the book should use this font.

File: styles/designs/carnival/parts/_appendix-components.scss [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+$AppendixTitleTextComponent: (
+  _name: "TitleText",
+  _subselector: '::before',
+  _properties: (
+    content: enum('ValueSet:::NULLABLE'),
+  )
+);
+
+$AppendixLayerOneComponent: (
+  _name: "Layer",
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_appendix-components.scss]: I am curious, why call this "layer"? When seeing this component, I first thought there would be some kind of background or z-index issue, but this contains only typography elements, it's unclear to me what exactly this component is supposed to be.

File: styles/designs/carnival/parts/_index-components.scss [hunk 9]
```diff
@@ -0,0 +1,91 @@
+// Learning Objectives Components
+$IndexContainerComponent: (
+  _name: "Container",
+  _subselector: '',
+  _properties: (
+    column-count: 3,
+    columns: 3,
+    column-gap: enum('ValueSet:::NULLABLE'),
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_index-components.scss]: Should this be nullable? If feels like we always would want a gap defined at all times. It should be modifiable but maybe not nullable?

File: styles/templates/template1/_settings.scss [hunk 10]
```diff
@@ -5,8 +5,8 @@
 @import 'templates/template1/parts/learning-settings.scss';
 @import 'templates/template1/parts/linktolearning-settings.scss';
 @import 'templates/template1/parts/portrait-settings.scss';
-@import 'templates/template1/parts/preface-settings';
-@import 'templates/template1/parts/_eoc-settings.scss';
+@import 'templates/template1/parts/appendix-settings.scss';
+@import 'templates/template1/parts/index-settings.scss';
```
Comment (helenemccarron) [inline styles/templates/template1/_settings.scss]: It looks like the preface and eoc settings have been removed, is this intentional?

File: styles/templates/template1/_typography.scss (hunk 5 above)
Comment (helenemccarron) [reply styles/templates/template1/_typography.scss]: Is this resolved? I can still see the value `paragraphText` in this commit.

File: styles/templates/template1/_settings.scss (hunk 10 above)
Comment (clee1060) [reply styles/templates/template1/_settings.scss]: No idea, I'll add them back... idk how they got deleted

**Pull request merged**